### DISCUSSION TITLE
Modbus Wiring Guide CT Images missing

### DISCUSSION THREAD

**Discussion by FozzieUK** [nathanmarlor/foxess_modbus#468] [Support]
The CT connector images no longer display in the modbus wiring guide, the 'assets' directory they refer to is no longer visible (EDIT: it is there but not visible for 'public' viewing - could the images be made public please (or moved to the images folder)

I don't have access to the image folder to upload and re-link the guide to them, so i've added them below - could @nathanmarlor or @canton7 copy to the images folder and re-link in the wiki please at https://github.com/nathanmarlor/foxess_modbus/wiki/Modbus-Wiring-Guide?fbclid=IwAR24QCaxhd6yE5OpXXCYJqXiTcWVsnCIZvnblnG6OJl1bgsWO6IldeyDebM#disassemble-clamp

Thanks

![Fox CT Connector1](https://github.com/nathanmarlor/foxess_modbus/assets/113460294/e051f449-c7d9-4cf5-95ef-a26f5c2dde72)
![Fox CT Connector2](https://github.com/nathanmarlor/foxess_modbus/assets/113460294/fdce28ea-d483-49c4-81f9-0adf06d5b363)
![Fox CT Connector3](https://github.com/nathanmarlor/foxess_modbus/assets/113460294/36aef8fb-1d64-40e5-aca6-d128c31a40bb)

**Comment by canton7** [comment]
How odd. I went to edit the page, and the old images showed up in the preview. Saved without making any changes, and they're back!

Additionally, there's no "assets" folder in the git repo which backs the wiki, or the main repo, and there never has been.

It's also weird that the other images are linked to user-images.githubusercontent.com, and I'll have uploaded them all in the same way, by pasting. The ones you've attached to this discussion have gone to private-user-images.githubusercontent.com.

I manually uploaded them to the git repo backing the wiki, and fixed the links. Hopefully that's the end of it!

**Reply by FozzieUK** [reply]
I'm sure they were there last week, I only noticed them missing today 🤷‍♂️ 

All working again now, as always many thanks for the quick response.
